Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BASE_VERSION=latest
+ARG BASE_VERSION=1.17.0
```

Comment:
Can still be overridden by doing a `--build-arg=latest` when building the browserless/chrome image.